WEBAPP-533: Supported browser (#364)
Integreat/integreat-webapp: docs/05-browser-stack.md

# Browser Stack
## Supported Browser
The Integreat Web-App has been tested on several OS environments with different browser clients.

> It can be used with most of the modern browsers that has been released up to the last six years.

Reviewer: ```suggestion
It can be used with most of the modern browsers that have been released in the last six years.
```
